T2/cluster_heimdall.py

```diff
@@ -6,15 +6,15 @@
 import numpy as np
 #from sklearn import cluster  # for dbscan
 import hdbscan
-from astropy import time, coordinates
+from astropy import time
 from astropy.io import ascii
 from astropy.io.ascii.core import InconsistentTableError
 try:
     from T2 import triggering
 except ModuleNotFoundError:
     print('not importing triggering')
 from event import names
-from dsautils import dsa_store
+from dsautils import dsa_store, coordinates
```

Comment:
Do we still need astropy.coordinates?

Reply:
Looks like we use the `coordinates.get_pointing` to calc the inital RA, Dec estimate.